classes/table/detail.php

```diff
@@ -65,7 +65,7 @@ public function __construct($baseurl, $taskid, $id = null) {
         parent::__construct('tool_lockstats_detail' . $id);
 
         $columns = array(
-            'task'  => get_string('table_resource', 'tool_lockstats'),
+            'classname'  => get_string('table_resource', 'tool_lockstats'),
```

Comment:
make this get_string('table_classname', 'tool_lockstats'), as we will use both strings in different places